src/mlpack/methods/linear_regression/linear_regression.cpp

```diff
@@ -107,14 +113,18 @@ void LinearRegression::Predict(const arma::mat& points,
   {
     // We want to be sure we have the correct number of dimensions in
     // the dataset.
-    Log::Assert(points.n_rows == parameters.n_rows);
+    util::CheckSameDimensionality(points, parameters, 
+        "LinearRegression::Predict()", "points");
```

Comment:
```suggestion
    util::CheckSameDimensionality(points, parameters, "LinearRegression::Predict()", "points");
```
I think that this will fit on one line.  (I didn't fully check, I'm just eyeballing it.)

Reply:
I made the following change so that we dont run out of 80-character line limit. 
Marcus pointed this out in his comment here.
[comment](https://github.com/mlpack/mlpack/pull/3164#discussion_r838985791).

Reply:
Ah, sorry, I didn't see that!  Nevermind my suggestion then. :+1: